src/ring_theory/ideals.lean

```diff
@@ -164,6 +164,58 @@ theorem is_coprime_self {x : α} :
   is_coprime x x ↔ is_unit x :=
 by rw [← span_singleton_eq_top]; simp [is_coprime]
 
+/- an indexed set of elements that are pairwise coprime -/
+def pairwise_coprime {α : Type u} [comm_ring α] {I : Type v} (s : I → α) : Prop :=
+∀ i j : I, i ≠ j → is_coprime (s i) (s j)
```

Comment:
We already have `pairwise` in `data.set.disjointed`, please use it. You'll get something like `pairwise is_coprime on s`. This is definitionally equal to this definition but gives you access to a few lemmas about `pairwise`.

Reply:
Note for the future that the syntax for docstrings uses `/--` and `-/`. (I had a comment to this effect which I deleted since it seems this def should be removed.0

Reply:
done